### PR TITLE
Laptop & PC Sprite Replacement

### PR DESCRIPTION
**Pull request by AndrewMontagne**
- Added new 3/4 perspective desktop and laptop computer sprites.
- Replaces laptop sprites for modular and security/medical laptop sprites
- Replaces library & stock market computer sprites

image

:cl: AndrewMontagne
add: New computer sprite for library & stock market computers.
add: New computer sprite for laptop computers.
add: Medical and security records desktop computer.
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/medical.dm [hunk 1]
```diff
@@ -603,9 +603,11 @@
...
 
 /obj/machinery/computer/med_data/laptop
-	name = "medical laptop"
-	desc = "A cheap Nanotrasen medical laptop, it functions as a medical records computer. It's bolted to the table."
-	icon_state = "laptop"
-	icon_screen = "medlaptop"
-	icon_keyboard = "laptop_key"
+	icon = 'icons/obj/modular_desktop.dmi'
+	name = "medical desktop"
+	desc = "A cheap Nanotrasen medical desktop, it functions as a medical records computer. It's bolted to the table."
```
Comment (FlattestGuitar) [inline code/game/machinery/computer/medical.dm]: It's not cheap, it's state-of-the-art technology.

Comment (AndrewMontagne) [reply code/game/machinery/computer/medical.dm]: State-of-the-art CRT desktop. 😝

**Comment by Birdtalon** [comment]
I like them, but I would prefer them in addition to the laptops and not as a direct replacement. Variety is the spice of life. We can even randomly choose whether to give laptops or computers down the line.

**Comment by Birdtalon** [comment]
Wrong button! 🙆‍♂️

**Comment by Birdtalon** [comment]
👍

**Comment by SamHPurp** [comment]
@FlattestGuitar Happy for this to merge pending your feedback.

**Comment by FlattestGuitar** [comment]
Most people seem to like this. Flavor!

:+1:

**Pull request merged**